deployll.sh

```diff
@@ -2121,7 +2121,7 @@ if [[ $LOCAL_INSTALL == true ]] && [[ $UPDATE_MODE == false ]]; then
     if [[ $CLAM_INSTALLED == true ]]; then
         sed -i "s?#CLAMDSCAN_BINARY=/usr/bin/clamscan?CLAMSCAN_BINARY=${CLAM_PATH}?" $LOCAL_PATH/${WEBAPP_SUBDIR}/.env
```

Comment:
@lukasz-learningpool noticed that this line also has a typo in the `sed` command
```suggestion
        sed -i "s?#CLAMDSCAN_BINARY=/usr/bin/clamscan?CLAMDSCAN_BINARY=${CLAM_PATH}?" $LOCAL_PATH/${WEBAPP_SUBDIR}/.env
```